src/autify/mobile/mobilelink/mobile-link-manager/MobileLinkManager.ts

```diff
@@ -66,6 +66,10 @@ export class MobileLinkManager {
     this.exec(["link", "setup"]);
   }
 
+  public async doctor(): Promise<void> {
```

Comment:
Not only for this command, but it may be nice to just pass the autify cli  arguments directly to mobilelink so that we don't need to care about autify cli side whenever we modify mobilelink arguments in the future.

Reply:
I want to show proper help if we have specific parameters. I wonder if proper help is compatible with pass-through 🤔

Reply:
Yeah, I confirmed the `this.argv` becomes empty because arguments are consumed when you run the `this.parse`.  Also, the `parse` fails if undefined parameters are given.

```
$ ./bin/dev mobile link start --aaa bcd
Error: Nonexistent flag: --aaa
```

So path argument varidation+help and path-through are not compatible.